pkg/destroy/aws/errortracker.go

```diff
@@ -0,0 +1,34 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	suppressDuration = time.Minute * 5
+)
+
+// ErrorTracker holds a history of errors
+type ErrorTracker struct {
+	history map[string]time.Time
```

Comment:
in don't think it's necessary today as the aws destroy is sequential..

but otherwise multiple invocation of suppressWarning that write to this map will create data races. Couldn't find a link in upstream docs, but here's a link 
https://stackoverflow.com/a/45258190
https://medium.com/a-journey-with-go/go-concurrency-access-with-maps-part-iii-8c0a0e4eb27e

Reply:
Good point. For now, I made the `errorTracker` object private.